packages/client3/tailwind.config.js

```diff
@@ -37,6 +37,9 @@ module.exports = {
           3: "#F4F4F4",
           4: "#FDFDFD",
         },
+        accent1: {
+          100: "#BFED47",
+        },
```

Comment:
I think @luansievers  may be merging his updated branch shortly, so potentially rebase off of his when that is done?

Reply:
Doesn't look like this is in his stuff though, soo... nevermind?